tools/osbuilder/node-builder/azure-linux/package_install.sh

```diff
@@ -40,6 +41,10 @@ if [ "${CONF_PODS}" == "yes" ]; then
 	echo "Installing SNP shim debug configuration"
 	cp -a --backup=numbered src/runtime/config/"${SHIM_DBG_CONFIG_FILE_NAME}" "${PREFIX}/${SHIM_CONFIG_PATH}"
 
+	if [ "${SHIM_USE_DEBUG_CONFIG}" == "yes" ]; then
```

Comment:
looks like we're pretty much discarding the original src/runtime/config/"${SHIM_CONFIG_FILE_NAME}" file, essentially replacing it with the debug config (see cp around line 63)...
 
...is that intended by this PR? Potentially we can still copy the debug config to its debug config location and then symlink the file to where containerd expects the CC shim config.  But: we should preserve the production/normal config as well (maybe even as simple as having this one as a .bak file?)

Maybe we can think of keeping the debug shim config a separate thing

Reply:
Let's add a comment here saying that the debug config will overwrite the release config.

Plus, to add use of variables in README

Reply:
Done